### PR TITLE
Clean-up sources

### PR DESCRIPTION
**Pull request by simonbrandhof**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [Periods.java#L89](https://github.com/SonarSource/sonarqube/blob/801d1a5f5167dc9541bb92633123803cb4dbb710/sonar-core/src/main/java/org/sonar/core/timemachine/Periods.java#L89): The Cyclomatic Complexity of this method "label" is 12 which is greater than 10 authorized. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
1. MINOR [DefaultUserFinder.java#L35](https://github.com/SonarSource/sonarqube/blob/801d1a5f5167dc9541bb92633123803cb4dbb710/server/sonar-server/src/main/java/org/sonar/server/user/DefaultUserFinder.java#L35): Remove this use of "UserFinder"; it is deprecated. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
1. MINOR [DeprecatedUserFinder.java#L31](https://github.com/SonarSource/sonarqube/blob/801d1a5f5167dc9541bb92633123803cb4dbb710/server/sonar-server/src/main/java/org/sonar/server/user/DeprecatedUserFinder.java#L31): Remove this use of "UserFinder"; it is deprecated. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: .gitattributes [hunk 1]
```diff
@@ -1 +0,0 @@
-sonar-server/src/test/resources/org/sonar/server/configuration/PropertiesBackupTest/backup-with-multiline-property.xml      eol=lf
```
Comment (sns-seb) [inline .gitattributes]: this change is unrelated to this commit

**Comment by sns-seb** [comment]
LGTM
